examples/app/BUILD.bazel

```diff
@@ -0,0 +1,54 @@
+load("@build_bazel_rules_nodejs//:defs.bzl", "http_server", "rollup_bundle")
+load("@npm_bazel_typescript//:index.bzl", "ts_config", "ts_devserver", "ts_library")
+
+ts_library(
```

Comment:
I was thinking that these would all go under `examples/typescript` showing how to use `packages/typescript`

otherwise if it's one-to-many, how would someone know where to find examples of using a particular package?

then the TypeScript example would have sub-packages showing the different features, like a protobuf under one subpackage. They could all be linked into the top-level example app.

WDYT?

Reply:
Hmmm. I think it's useful to have separate workspaces for each example. Keeps it self contained and the package.json is only for that example and a good example really shouldn't need to be updated very often. However, you're right that having them all at the top level may be too much but some examples use multiple packages such as `typescript` and `karma` and `jasmine`.

Maybe just a good `examples/README.md` that provides an overview of what each example shows?